data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/petrifies/petrifies_ability.cfg

```diff
@@ -0,0 +1,108 @@
+#textdomain wesnoth-test
+
+#####
+# API(s) being tested: [petrifies]
+##
+# Actions:
+# Give the leaders a petrifies ability
+# Have side 1's leader attack side 2's leader with both their weapons
+##
+# Expected end state:
+# Both weapons petrify side 2's leader
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "petrifies_ability" (
+    [event]
+        name = start
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY_NO_VALUE petrifies () SELF=yes}
+                [/abilities]
+            [/effect]
+        [/modify_unit]
+
+        [modify_unit]
+            [filter]
+                id=alice,bob
+            [/filter]
+            [effect]
+                apply_to=attack
+                [set_specials]
+                    mode=append
+                    [attacks]
+                        value=1
+                    [/attacks]
+                    [damage]
+                        value=1
+                    [/damage]
+                    [chance_to_hit]
+                        value=100
+                    [/chance_to_hit]
+                [/set_specials]
+            [/effect]
+        [/modify_unit]
+
+        [test_do_attack_by_id]
+            attacker=alice
+            defender=bob
+            weapon=0
+            resupply_attacks_left=1
+        [/test_do_attack_by_id]
+
+        {ASSERT (
+            [have_unit]
+                side = 2
+                status = "petrified"
+            [/have_unit]
+        )}
+        {ASSERT (
+            [not]
+                [have_unit]
+                    side = 1
+                    status = "petrified"
+                [/have_unit]
+            [/not]
+        )}
+
+        [unpetrify]
+            side = 3
```

Comment:
There doesn't seem to be a side 3 in this test

Reply:
Updated.